packages/next/build/index.ts

```diff
@@ -525,6 +529,40 @@ export default async function build(dir: string, conf = null): Promise<void> {
   )
   staticCheckWorkers.end()
 
+  if (serverPropsPages.size > 0) {
+    // We update the routes manifest after the build with the
+    // serverProps routes since we can't determine this until after build
+    routesManifest.serverPropsRoutes = {}
+
+    for (const page of serverPropsPages) {
+      const cleanDataRoute = path.posix.join(
```

Comment:
Let's move this variable closer to usage.

Reply:
Updated